src/components/side_panel/chart/line_chart/line_chart_config_panel.xml

```diff
@@ -14,6 +14,18 @@
             Stacked linechart
           </label>
         </div>
+        <div class="o-checkbox">
+          <label>
+            <input
+              type="checkbox"
+              name="cumulated"
+              t-att-checked="props.definition.cumulated"
+              t-on-change="onUpdateCumulated"
+              class="align-middle"
+            />
+            Cumulated Data
```

Comment:
```suggestion
            Cumulated data
```
to stay consistent with the other options

Reply:
> to stay consistent with the other options
sure, done.